src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -279,12 +272,7 @@ private static ArgumentParser getArgumentParser() {
                 .help(FeatureControl.SUPPRESS);
 
         // Testing flags
-        argumentGroup.addArgument(TEST_MODE_FLAG)
-                .dest(TEST_MODE_FLAG[0])
-                .action(Arguments.storeTrue())
-                .help("Enables testing mode.");
-
-        argumentGroup.addArgument(FRESH_CLONING_FLAG)
+        parser.addArgument(FRESH_CLONING_FLAG)
                 .dest(FRESH_CLONING_FLAG[0])
                 .action(Arguments.storeTrue())
                 .help("Enables fresh cloning. Requires testing mode to be enabled.");
```

Comment:
```suggestion
                .help("Enables fresh cloning.");
```

Now we don't have testing mode, I think we can remove this part.